tests/category_test.go

```diff
@@ -104,8 +105,8 @@ func TestMarkCategoryAsRead(t *testing.T) {
 	if results.Total == 0 {
 		t.Fatalf(`Invalid number of entries: %d`, results.Total)
 	}
-	if results.Entries[0].Status != miniflux.EntryStatusUnread {
-		t.Fatalf(`Invalid entry status, got %q instead of %q`, results.Entries[0].Status, miniflux.EntryStatusUnread)
+	if results.Entries[0].Status != model.EntryStatusUnread {
+		t.Fatalf(`Invalid entry status, got %q instead of %q`, results.Entries[0].Status, model.EntryStatusUnread)
```

Comment:
The package `tests` are the integration tests. These tests are not sharing code with the rest of the application. As such, it should keep the constants provided by the official client.